### DISCUSSION TITLE
引入了"@umijs/plugins/dist/antd"，但import时'antd'报错

### DISCUSSION THREAD

**Discussion by Zeyull** [umijs/umi#12492] [Q&A]
按照官网文档安装了@umijs/plugins
`pnpm add -D @umijs/plugins`
然后在.umirc.ts中也开启了antd配置
`plugins: ['@umijs/plugins/dist/antd'],`
但是在引用后报错，无法找到 module 'antd'
![image](https://github.com/umijs/umi/assets/50110204/c4201db6-3496-4c55-a575-36db043c8833)
请问这报错如何修复？
**虽然报错，但本地启动项目也可以正常运行**

我知道`pnpm i antd`可以解决问题，但是我明明可以用umi中的插件，为什么还要单独再引入antd呢

**Comment by fz6m** [comment]
`@umijs/plugins` 自带了一个 antd 依赖，如果你不想安装，就会默认用内置自带的，如果想用自己的，就显示安装一个 antd 。

但因为你使用的是 pnpm ，默认情况是一个严格隔离的 node_modules ，所以没安装的依赖就是索引不到，不允许幽灵依赖，必须显示安装；只有用打平的包管理器如 yarn v1 / npm 才可以使用幽灵依赖。
